src/controller.c

```diff
@@ -22,7 +25,7 @@ int32_t sat(int32_t value, int32_t max)
 
 void run_controller(uint32_t now)
 {
-  control_t outgoing_command = rate_controller(attitude_controller(_combined_control, now), now);
+  control_t outgoing_command = rate_controller(_combined_control, now);
```

Comment:
Until I have the rate controller working, I'm just ignoring the attitude controller